linchpin/provision/library/os_server2.py

```diff
@@ -762,5 +789,8 @@ def main():
         module.fail_json(msg=str(e), extra_data=e.extra_data)
 
 
+# this is magic, see lib/ansible/module_common.py
+from ansible.module_utils.basic import *
```

Comment:
Take these imports out as they are already at the top. Also, importing * is not recommended. Please remove this and import ONLY what is needed. I know that this is how Ansible does it, but it doesn't appear to add any value.